proc_info.sh

```diff
@@ -15,3 +15,76 @@ function usage(){
   echo "-f                   Show opened files by the process"
   echo "-h                   Show this help"
 }
+
+#Function for -u flag
+get_owner(){
+  pid=$1
+  user_id=$(cat /proc/$pid/status | grep Uid | awk '{ print $2 }')
+  #https://unix.stackexchange.com/a/36582/468165 - Get user by Id
+  getent passwd "$user_id" | cut -d: -f1
+}
+
+#Function for -c flag
+get_command(){
+  pid=$1
+  #cat /proc/$pid/cmdline ; echo
+  IFS=''
+  cat /proc/$pid/cmdline |
+  while read data
+  do
+    echo "$data"
+  done
+}
```

Comment:
After update I got this:
```
vagrant@wordpress:~$ sleep 1000 &
[3] 2141
vagrant@wordpress:~$ /vagrant/proc_info.sh 2141 -c
vagrant@wordpress:~$
```

Reply:
I'm getting the same behavior! Gonna have to fix that, thanks for letting me know.

Reply:
BTW. I'm testing it on VM created by Vagrant. So it's should be always same result.